dokka-subprojects/plugin-base/src/test/kotlin/model/ConstructorsTest.kt

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package model
+
+import org.jetbrains.dokka.analysis.kotlin.markdown.MARKDOWN_ELEMENT_FILE_NAME
+import org.jetbrains.dokka.model.*
+import org.jetbrains.dokka.model.doc.*
+import org.jetbrains.dokka.model.doc.P
+import kotlin.test.Test
+import utils.*
+
+
+class ConstructorsTest : AbstractModelTest("/src/main/kotlin/constructors/Test.kt", "constructors") {
```

Comment:
> What do you think about adding additional tests for the documentable model?

These tests are already for the documentable model. Here the `model` is the documentable one, no content model.

> Model tests are OKish since they transitively test the analysis

What do you mean `transitively test`?